### ISSUE TITLE
Cyberpunk 2077 flickering artifacts on flying particles.

### ISSUE THREAD

**Issue by huyanh138** [Nukem9/dlssg-to-fsr3#104]
In Cyberpunk 2077, especially in Dogtown, the flying news paper seem to be flickering heavily with frame generation enabled. I'm using 3080 10GB and with FG on I average 50-60 FPS in Dogtown and 70-80 elsewhere. DLSS quality max RT and other settings.

**Comment by T-vonGodefroy** [comment]
yep, i was testing the performance in dogtown with PT and it was very noticeable
